src/Combobox.js

```diff
@@ -64,7 +65,12 @@ var Combobox = React.createClass({
      * The component to render for the list in the popup.
      * Default is `List`.
      */
-    listComponent: React.PropTypes.func
+    listComponent: React.PropTypes.func,
+
+    /**
+     * The number of milliseconds to throttle `getOptionsForInputValue()`.
+     */
+    getOptionsThrottleMs: React.PropTypes.number,
```

Comment:
Can we make this `optionsGetterThrottleMs`? I initially thought this prop was supposed to be a function returning the number of seconds to throttle for, instead of just a number of seconds to throttle for.